elpy.el

```diff
@@ -537,6 +539,8 @@ virtualenv.
   :lighter " Elpy"
   (when (not (derived-mode-p 'python-mode))
     (error "Elpy only works with `python-mode'"))
+  (when (featurep 'xref)
+    (add-hook 'xref-backend-functions #'elpy--xref-backend nil t))
```

Comment:
Also, I think this is missing the conditional definition of `M-.` and `M-,` to use the xref functionality … or am I missing something? I.e. if xref is available, use that instead of the old elpy functionality?